src/main/api/apigClient.js

```diff
@@ -53,7 +53,7 @@ apigClientFactory.newClient = function (config) {
 
     
     // extract endpoint and path from url
-    var invokeUrl = 'https://c72uud7t8a.execute-api.us-west-1.amazonaws.com/beta';
+    var invokeUrl = 'https://rqah8n9kv6.execute-api.us-west-2.amazonaws.com/beta';
```

Comment:
After you've updated apigClient.js please make sure this line still says:
`var invokeUrl = config.invokeUrl;`

And update the actual url in:
`src/main/ionic/peers/src/environments/environment.ts`